app-misc/anki/anki-23.10.1.ebuild

```diff
@@ -0,0 +1,801 @@
+#
+# Anki bundles 3rd-party code and assets:
+# The MathJax files are licensed under Apache-2.0.
+# - node_modules/mathjax/es5/
+#
+# CSS Browser Selector is licensed under CC-BY-2.5.
+# - node_modules/css-browser-selector/css_browser_selector.min.js
+#
+# The fancy deboss pattern is licensed under CC-BY-4.0.
+# - pylib/anki/statsbg.py
+#
+# The patched MPV controls are licensed under MIT.
+# - qt/aqt/mpv.py
+#
+# The Winpath module is licensed under MIT.
+# - qt/aqt/winpaths.py
+#
+# The licenses for the runtime JS libaries are documented in the source code.
+# - ts/licenses.json
+# How to get an up-to-date summary:
+# ./node_modules/.bin/license-checker-rseidelsohn --production --excludePackages anki --summary
+#
+# The vendored Flot plotting library is licensed under MIT.
+# - qt/aqt/data/web/js/vendor/plot.js
+LICENSE+=" !minimal? ( 0BSD CC-BY-4.0 CC-BY-SA-2.5 GPL-3+ Unlicense )"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+REQUIRED_USE="!minimal? ( ${PYTHON_REQUIRED_USE} )"
+
+# Dependencies:
+# Python: python/requirements.{anki,aqt}.in
+# If ENABLE_QT5_COMPAT is set at runtime
+# additionally depend on PyQt6[dbus,printsupport].
+# Qt: qt/aqt/{sound.py,qt/*.py}
+# app-misc/certificates: The rust backend library is built against
+# rustls-native-certs to use the native certificate store.
+
+DEPEND="
+	dev-db/sqlite:3
+	>=app-arch/zstd-1.5.5:=
+"
+GUI="
+	$(python_gen_cond_dep '
+		dev-python/beautifulsoup4[${PYTHON_USEDEP}]
+		dev-python/distro[${PYTHON_USEDEP}]
+		dev-python/decorator[${PYTHON_USEDEP}]
+		dev-python/flask[${PYTHON_USEDEP}]
+		dev-python/flask-cors[${PYTHON_USEDEP}]
+		dev-python/jsonschema[${PYTHON_USEDEP}]
+		dev-python/markdown[${PYTHON_USEDEP}]
+		dev-python/protobuf-python[${PYTHON_USEDEP}]
+		dev-python/requests[${PYTHON_USEDEP}]
+		dev-python/send2trash[${PYTHON_USEDEP}]
+		dev-python/waitress[${PYTHON_USEDEP}]
+	')
+	qt6? (   >=dev-qt/qtsvg-6.5:6
+			 $(python_gen_cond_dep '
+			   >=dev-python/PyQt6-6.5.0[gui,network,opengl,quick,webchannel,widgets,${PYTHON_USEDEP}]
+			   >=dev-python/PyQt6-sip-13.5.1[${PYTHON_USEDEP}]
+			   >=dev-python/PyQt6-WebEngine-6.5.0[widgets,${PYTHON_USEDEP}]')
+
+		 )
+	!qt6? ( >=dev-qt/qtgui-5.15:5[jpeg,png]
+			>=dev-qt/qtsvg-5.15:5
+			$(python_gen_cond_dep '
+				>=dev-python/PyQt5-5.15.5[gui,network,webchannel,widgets,${PYTHON_USEDEP}]
+				>=dev-python/PyQt5-sip-12.9.0[${PYTHON_USEDEP}]
+				>=dev-python/PyQtWebEngine-5.15.5[${PYTHON_USEDEP}]')
+
+		  )
+	${PYTHON_DEPS}
+"
+RDEPEND="
+	app-misc/ca-certificates
+	!minimal? ( ${GUI} )
+	${DEPEND}
+"
+BDEPEND="
+	dev-libs/protobuf
+	virtual/pkgconfig
+	>=virtual/rust-1.71.0
+	!minimal? (
+			  app-arch/unzip
+			  dev-util/ninja
+			  net-libs/nodejs
+			  $(python_gen_cond_dep 'dev-python/wheel[${PYTHON_USEDEP}]')
+			  qt6? ( $(python_gen_cond_dep '>=dev-python/PyQt6-6.5.0[${PYTHON_USEDEP}]') )
+			  !qt6? ( $(python_gen_cond_dep '>=dev-python/PyQt5-5.15.5[${PYTHON_USEDEP}]') )
+			  ${PYTHON_DEPS}
+			  )
```

Comment:
Fix indentation

Reply:
https://github.com/gentoo/gentoo/blob/90530f31413ac1558e36b47123186b392c451fc9/app-misc/anki/anki-23.12.1.ebuild#L791-L792
Emacs' *DEPEND indentation started to get frustrating so I switched to vim. I hope the new indentation is ok.